src/System.Diagnostics.EventLog/src/System/Diagnostics/UnsafeNativeMethods.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+namespace Microsoft.Win32
+{
+    internal static class UnsafeNativeMethods
+    {
```

Comment:
these methods should go into the Interop folder as described in the PInvoke guidelines. @danmosemsft correct?

Reply:
I suggested if they are EventLog specific they remain here since they're not going to be used by anyone else.